nebula-logger/main/logger/aura/logger/loggerHelper.js

```diff
@@ -1,55 +1,55 @@
 /*************************************************************************************************
-* This file is part of the Nebula Logger project, released under the MIT License.                *
-* See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    *
-*************************************************************************************************/
+ * This file is part of the Nebula Logger project, released under the MIT License.                *
+ * See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    *
+ *************************************************************************************************/
 ({
-    addEntry : function(component, event) {
+    addEntry: function (component, event) {
         var timestamp = new Date().toISOString();
 
         var logEntries = component.get('v.logEntries');
-        if(logEntries == undefined || logEntries == null) logEntries = [];
+        if (logEntries == undefined || logEntries == null) logEntries = [];
```

Comment:
To `{ }` or not to `{ }` -  that is the question
```
if (logEntries == undefined || logEntries == null) {
    logEntries = [];
}
```

Reply:
this one I just went with the default